### ISSUE TITLE
Error with HelveticaNeue font-family

### ISSUE THREAD

**Issue by lucj** [MosesEsan/react-native-basic-form#9]
Is it possible to use another font-family than the HelveticaNeue on Android ? It raises the following error on a Mi Note 6 pro:

```
"HelveticaNeue-Medium" is not a system font and has not been loaded through Font.loadAsync.
```

**Comment by lucj** [comment]
Just commenting the following line in Form.js fixed the thing.

````
buttonText: {
        fontSize: 18,
        color: "#fff",
        //fontFamily: "HelveticaNeue-Medium"
    }
````

**Comment by MosesEsan** [comment]
Commit 0e61e9d94820a0ad39ad4ce90ee9fab5240b4937 closes #9

**Comment by MosesEsan** [comment]
Released in v1.1.11
